src/Private/OpenApi.ps1

```diff
@@ -124,6 +124,14 @@ function ConvertTo-PodeOASchemaProperty
         $Property
     )
 
+    # schema refs
+    if($Property['$ref']) {
```

Comment:
This gets mentioned below, but if we add a `type` to the hashtable, this could be the following instead:

```powershell
if ($Property.type -ieq 'schema') {
```

Reply:
I've done this and also allowed it to be an array.